mylyn.context/org.eclipse.mylyn.context.tasks.ui/src/org/eclipse/mylyn/internal/context/tasks/ui/editors/ContextEditorFormPage.java

```diff
@@ -231,20 +231,27 @@ public void dispose() {
 		}
 	}
 
+	private int setNewMaxWith(int oldValue, Control imageControl, Control linkControl) {
+		Point imageSize = imageControl.computeSize(SWT.DEFAULT, SWT.DEFAULT, true);
+		Point hyperlinkSize = linkControl.computeSize(SWT.DEFAULT, SWT.DEFAULT, true);
+		return imageSize.x + hyperlinkSize.x > oldValue ? imageSize.x + hyperlinkSize.x : oldValue;
```

Comment:
Would you consider to extract `imageSize.x + hyperlinkSize.x` to a local variable (and inline `imageSize` and `hyperlinkSize`)?
In this case return statement will be much more readable:
```java
return required > existing ? required : existing
```